webrtc.html

```diff
@@ -4106,7 +4106,7 @@
               the candidate that it is derived from. For host candidates, the
               <code>relatedPort</code> is <code>null</code>. This corresponds to
               the <code>rel-port</code> field in <a><code>candidate-attribute</code></a>.</dd>
-              <dt><code><dfn>ufrag</dfn></code> of type <span class=
+              <dt><code><dfn>usernameFragment</dfn></code> of type <span class=
               "idlAttrType"><a>DOMString</a></span>, readonly, nullable</dt>
               <dd>This carries the <code>ufrag</code> as defined in section
```

Comment:
"ice-ufrag" is the attribute, "ufrag" is the field inside that attribute. So I think we're ok.